python/tests/contrib/torchhub/test_resnet.py

```diff
@@ -0,0 +1,44 @@
+#  Copyright 2021 Rikai Authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from pathlib import Path
+
+import torchvision
+from pyspark.sql import SparkSession
+
+from rikai.spark.functions import to_image
+from rikai.spark.utils import init_spark_session
+
+
+def test_resnet():
+    spark = init_spark_session()
```

Comment:
must use the separate spark session, otherwise, python path will be polluted

Reply:
what pollutes the python path in this test?

Reply:
`torch.hub.load()` will add the download repo into the import path

Reply:
Dont understand why `load()` downloads is a bad thing here. Can you elabrate?

Reply:
This is the failed CI: https://github.com/eto-ai/rikai/runs/4735497844?check_suite_focus=true